### DISCUSSION TITLE
plugin-fullscreen aborts after switching to fullscreen (jsPsych 7.0)

### DISCUSSION THREAD

**Discussion by mmbcorley** [jspsych/jsPsych#2300] [Q&A] [answered]
Hi, switching to fullscreen works but the experiment stops at that point and nothing further is shown.  The console shows
```
Uncaught TypeError: Cannot read properties of undefined (reading 'finishTrial')
    at plugin-fullscreen@1.0.0:1
```
I've put a minimal example (shows instruction screen, switches to fullscreen, should show instruction screen again) [online](https://www.corleylab.ppls.ed.ac.uk/gano1/minimal.html).  I'd be grateful for any pointers!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @mmbcorley,

There was a bug in the 1.0.0 version of this plugin. We [released version 1.0.1](https://github.com/jspsych/jsPsych/releases/tag/%40jspsych%2Fplugin-fullscreen%401.0.1), which should fix it.
